WalletWasabi.Gui/CommandLine/CommandInterpreter.cs

```diff
@@ -1,22 +1,27 @@
 using Mono.Options;
 using System;
-using System.IO;
 using System.Linq;
-using System.Runtime.InteropServices;
 using System.Threading.Tasks;
 using WalletWasabi.Helpers;
-using WalletWasabi.Logging;
 
 namespace WalletWasabi.Gui.CommandLine
 {
-	public static class CommandInterpreter
+	public class CommandInterpreter
 	{
+		private readonly PasswordFinderCommand PasswordFinderCommand;
```

Comment:
please use read only properties instead of fields.

Reply:
Ok. Fixed.